src/sentry/features/__init__.py

```diff
@@ -53,36 +53,36 @@
-default_manager.add("organizations:sso-basic", OrganizationFeature)  # NOQA
-default_manager.add("organizations:sso-rippling", OrganizationFeature)  # NOQA
-default_manager.add("organizations:sso-saml2", OrganizationFeature)  # NOQA
-default_manager.add("organizations:grouping-info", OrganizationFeature)  # NOQA
-default_manager.add("organizations:tweak-grouping-config", OrganizationFeature)  # NOQA
-default_manager.add("organizations:set-grouping-config", OrganizationFeature)  # NOQA
+default_manager.add('organizations:advanced-search', OrganizationFeature)  # NOQA
+default_manager.add('organizations:boolean-search', OrganizationFeature)  # NOQA
+default_manager.add('organizations:api-keys', OrganizationFeature)  # NOQA
+default_manager.add('organizations:discover', OrganizationFeature)  # NOQA
+default_manager.add('organizations:events', OrganizationFeature)  # NOQA
+default_manager.add('organizations:events-v2', OrganizationFeature)  # NOQA
+default_manager.add('organizations:event-attachments', OrganizationFeature)  # NOQA
+default_manager.add('organizations:symbol-sources', OrganizationFeature)  # NOQA
+default_manager.add('organizations:global-views', OrganizationFeature)  # NOQA
+default_manager.add('organizations:incidents', OrganizationFeature)  # NOQA
+default_manager.add('organizations:integrations-issue-basic', OrganizationFeature)  # NOQA
+default_manager.add('organizations:integrations-issue-sync', OrganizationFeature)  # NOQA
+default_manager.add('organizations:integrations-event-hooks', OrganizationFeature)  # NOQA
+default_manager.add('organizations:internal-catchall', OrganizationFeature)  # NOQA
+default_manager.add('organizations:incidents', OrganizationFeature)  # NOQA
+default_manager.add('organizations:invite-members', OrganizationFeature)  # NOQA
+default_manager.add('organizations:large-debug-files', OrganizationFeature)  # NOQA
+default_manager.add('organizations:legacy-event-id', OrganizationFeature)  # NOQA
+default_manager.add('organizations:monitors', OrganizationFeature)  # NOQA
+default_manager.add('organizations:onboarding', OrganizationFeature)  # NOQA
+default_manager.add('organizations:org-saved-searches', OrganizationFeature)  # NOQA
+default_manager.add('organizations:relay', OrganizationFeature)  # NOQA
+default_manager.add('organizations:require-2fa', OrganizationFeature)  # NOQA
+default_manager.add('organizations:sentry10', OrganizationFeature)  # NOQA
+default_manager.add('organizations:sso-basic', OrganizationFeature)  # NOQA
+default_manager.add('organizations:sso-rippling', OrganizationFeature)  # NOQA
+default_manager.add('organizations:sso-saml2', OrganizationFeature)  # NOQA
+default_manager.add('organizations:grouping-info', OrganizationFeature)  # NOQA
+default_manager.add('organizations:tweak-grouping-config', OrganizationFeature)  # NOQA
+default_manager.add('organizations:set-grouping-config', OrganizationFeature)  # NOQA
```

Comment:
Was this done automatically or did you change this somehow?

Reply:
@mnoble automatically after I removed the feature flag we care about